actor/src/main/scala/org/apache/pekko/dispatch/ForkJoinPoolConstants.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.pekko.dispatch
+
+private[dispatch] object ForkJoinPoolConstants {
+  final val MaxCap: Int = 0x7FFF // 32767
+  final val DefaultKeepAlive: Long = 60000
```

Comment:
DefaultKeepAliveInMiilis

Reply:
done

Reply:
Thank you, but Take care, time to sleep 😴 💤 😪